frontend/src/Components/BookingPage/BookingPage.jsx

```diff
@@ -9,6 +9,22 @@ const BookingPage = () => {
     const [date, setDate] = useState('');
     const [time, setTime] = useState('');
     const [description, setDescription] = useState('');
+    const [services, setServices] = useState([]);
+
+
+    useEffect(() => {
+        const fetchServices = async () => {
+            try {
+                const response = await fetch(`http://localhost:5174/service-provider-services/${providerId}`);
```

Comment:
noob question: do we need to hardcode this? If we are always going to have the same port then that's fine

Reply:
I don't think so. I will update it in subsequent PR's. Thank You